tests/test_operation.py

```diff
@@ -123,6 +123,9 @@ def test_heisenberg(self, test_class, tol):
     def test_operation_init(self, test_class, monkeypatch):
         "Operation subclass initialization."
 
+        if test_class == qml.ControlledQubitUnitary:
+            pytest.skip("ControlledQubitUnitary alters the input params and wires in its __init__")
```

Comment:
Would be good to swap it with a dedicated `__init__` test case. `PauliRot` also seems to have  `__init__`, how come that's not skipped? :thinking:

Reply:
Added a test